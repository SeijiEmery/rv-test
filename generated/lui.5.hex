0000000    b7  f5  ff  ff  13  c6  f5  7f  73  00  10  00  13  00  00  00
0000020    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    73  00  10  00                                                
0000044
