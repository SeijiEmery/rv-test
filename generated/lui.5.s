lui a1, 0xFFFFF
    xori a2, a1, 0x7FF # can't actually store 12 bit value cuz signed! argh....
    nop
    nop
    nop
    nop
    ebreak
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
