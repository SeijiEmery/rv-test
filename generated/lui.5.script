load /x 72 generated/lui.5.hex
writereg -1 72
writereg 11 0
setpc 72
run 7
readreg -1
readreg 11
readreg 12