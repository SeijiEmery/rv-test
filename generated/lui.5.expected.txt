tests/lui.test.s:39
pc = 80
R11 = 18446744073709547520
R12 = 18446744073709549567
